src/aaudio/AAudioExtensions.h

```diff
@@ -80,12 +142,22 @@ class AAudioExtensions {
      * @return 0 or a negative error code
      */
     int32_t setMMapEnabled(bool enabled) {
+        if (mLibLoader->aaudio_setMMapPolicy != nullptr) {
```

Comment:
Can you add a comment for this file explaining why there are 2 cases, one when the APIs are officially supported and one when they arent?

Reply:
Done